htmd/__init__.py

```diff
@@ -1,11 +1,17 @@
 from htmd.ui import *  # TODO: This will be deprecated in a few versions
-from htmd.config import config
 from htmd.version import version as _version
+from htmdx.cli import check_registration, show_news
+from htmd.latest import compareVersions
 import os
 
 config()
 __version__ = _version()
 
+if not (os.getenv("HTMD_NONINTERACTIVE")):
```

Comment:
Also registration check should be in ui to avoid unnecessary slowdown and messages when importing directly